exporter/datadogexporter/internal/testutil/test_utils.go

```diff
@@ -45,16 +45,26 @@ type DatadogServer struct {
 	MetadataChan chan []byte
 }
 
+/* #nosec G101 */
```

Comment:
Could you add a comment justifying the `nosec` directive? (I am fine with it, but want to have the rationale recorded in-code)